pkgs/development/web/cypress/default.nix

```diff
@@ -35,10 +36,11 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    description = "Fast, easy and reliable testing for anything that runs in a browser";
+    description =
+      "Fast, easy and reliable testing for anything that runs in a browser";
```

Comment:
```suggestion
    description = "Fast, easy and reliable testing for anything that runs in a browser";
```

Reply:
nixfmt undoes the single-line version, maybe because it's wider than 80 characters.
I'd prefer to keep the formatting consistent with nixfmt, and use the multiline version; is there any official advice for or against using nixfmt in nixpkgs?

Reply:
> nixfmt undoes the single-line version, maybe because it's wider than 80 characters.

I don't agree with 80 characters line hard limit. This just adds useless new lines and updates should not really reformat everything because a formatter did it.

> I'd prefer to keep the formatting consistent with nixfmt

There is generally nothing wrong with this but some changes it does like the on above just make the code harder to read.

> is there any official advice for or against using nixfmt in nixpkgs?

Generally it is good to use but it also does weird things like this. I would suggest to use nixpkgs-fmt but it has its flaws, too.

Reply:
No worries, I've [redone the style commit](https://github.com/NixOS/nixpkgs/compare/4bff8bc4535c7639759113612aeb0bed9d5b1b51..758337b0d8e07987745e687b940d4fe5c6a13117) to keep it on one line.